### PR TITLE
Store processing sequence id on Basic protobuf

### PR DESCRIPTION
**Pull request by jotare**

### CODE REVIEW

File: nucliadb_writer/nucliadb_writer/resource/basic.py [hunk 1]
```diff
@@ -213,3 +213,7 @@ def set_status(basic: Basic, item: CreateResourcePayload):
 
 def set_status_modify(basic: Basic, item: UpdateResourcePayload):
     basic.metadata.status = Metadata.Status.PENDING
+
+
+def set_seqid(bm: BrokerMessage, seqid: int):
```
Comment (lferran) [inline nucliadb_writer/nucliadb_writer/resource/basic.py:218]: to be more accurate: `set_last_seqid`?

**Review by lferran — APPROVED** [review]
lgtm! 👍🏼

File: nucliadb_writer/nucliadb_writer/resource/basic.py (hunk 1 above)
Comment (jotare) [reply nucliadb_writer/nucliadb_writer/resource/basic.py]: Changed

**Review by lferran — COMMENTED** [review]
I think you are still missing some cases, like when updating a field text, a field link, and others.

**Pull request merged**